lessons/codap-transformers2/langs/en-us/pages/codap-sort-transformer.adoc

```diff
@@ -6,29 +6,31 @@
 </style>
 ++++
 
-Make sure you’re logged into CODAP with the Animals Dataset open. Open the Transformers plugin, and select `Sort`.
+Make sure you’re logged @starter-file{animals}. Select the "Plugins" icon, and choose "Transformers."
 
 @vspace{1ex}
 
 == Creating a Sort Transformer
 
-- Select `Sort` from the drop-down menu of Transformers.
-- We want to give each Transformer a useful name, one which clearly describes what that Transformer does. Generally, a brief hyphenated description works - for instance: `order-by-pounds`.
+@n We're going to create a Transformer to sort the animals alphabetically by name. Select Sort from the drop-down menu.
 
-@n What would be a good name for a Transformer that organizes the table alphabetically by animals' names?
+@n What would be a good name for a Transformer that organizes the table alphabetically by animals' names? @fitb{}{@ifsoln{order-by-name.}}
 
-@fitb{}{@ifsoln{order-by-name.}}
-
-- Type in the name you wrote above.
+_Type in that name._
```

Comment:
Where? If you're going to take up a whole line of text to give explicit instructions, let's use the whole line and be super-clear. Heck, maybe a screenshot of a transformer would be helpful on this page, with the fields numbered? Then you could say "type that name into box 1". 

Then rinse and repeat for the other transformer pages